app/lib/mammoth/channels.rb

```diff
@@ -64,7 +68,7 @@ def account_ids(accounts)
     # Channel includes id, title, description, owner
     def list(include_accounts: false)
       cache_key = include_accounts ? 'channels:list:w_accounts' : 'channels:list'
-      Rails.cache.fetch(cache_key, expires_in: 60.seconds) do
+      Rails.cache.fetch(cache_key, expires_in: 1.hour) do
```

Comment:
Just confirming this change from 1 min to 1 hour is expected.

Reply:
yup. I wanted to keep the channels and the accounts that we've added to them cached, but when they are edited not have to wait more than an hour for them to populate